cli/stop.go

```diff
@@ -46,10 +46,11 @@ func (s *StopCommand) runStop(args []string) error {
 	ctx := context.Background()
 	apiClient := s.cli.Client()
 
-	container := args[0]
-
-	if err := apiClient.ContainerStop(ctx, container, strconv.Itoa(s.timeout)); err != nil {
-		return fmt.Errorf("failed to stop container %s: %v", container, err)
+	for _, name := range args {
+		if err := apiClient.ContainerStop(ctx, name, strconv.Itoa(s.timeout)); err != nil {
+			return fmt.Errorf("failed to stop container %s: %v", name, err)
```

Comment:
It is a good work. While I am wondering if it is reasonable to quick return when meeting one container stop error. For example:
```
pouch stop will-error normal0 normal1
```
if the first container stop returns an error, then the next two container will not receive the stop request.

How about just output the error message for the error container in the stderr and make the while command line return a exit code non-zero?